src/test/resources/application.yml

```diff
@@ -68,9 +68,32 @@ authenticator:
         fieldsToExtract:
           firstName: name.split(' ')[0]
           lastName: name.split(' ')[1]
-
+    ldap:
+      service: org.ohdsi.authenticator.service.directory.ldap.LdapAuthService
+      config:
+        baseDn: ou=users,dc=example,dc=com
+        userDn: uid=admin,ou=system
+        password: secret
+        url: ldap://localhost:8389
+        searchFilter: uid={0}
+        countLimit: 0
+        ignorePartialResultException: true
+        fieldsToExtract:
+          firstName: displayName
+          lastName: sn
 security:
   jwt:
     token:
       secretKey: 129DF19C8A91AFD8375A2826A33539K01ACQ778QOJFAA9MGWLWH73PLXVFVHBR7860MTIE2O8EEVF9KCO77P6A7NUNX4XHAGCRFSBWG879XPDOIN6C2LFCKJI002OIABS4D6Q9VMJJIX8UCE48EF
-      validityInSeconds: 60
\ No newline at end of file
+      validityInSeconds: 60
+spring:
```

Comment:
Again, to keep things consistent, why not to put it into separate `application-test-ldap.yml` as with AD?